docs/guides/bugzilla.rst

```diff
@@ -0,0 +1,192 @@
+Bugzilla Guide
+==============
+
+Contributor guidelines
+----------------------
+We have developed several tools/workflows to make the process
+for dealing with Bugzilla bugs (BZ's) easier for contributors.
+Here, we discuss:
+
+* Metadata markers for test cases: ``blockers``, ``coverage``, and ``automates``
+* the ``miq bz`` command
+* ``Bugzilla:`` docblocks
+
+Metadata Markers
+^^^^^^^^^^^^^^^^
+* **blockers**: this marker is used when a test case is `blocked` by a bugzilla bug.
+  To use this marker simply add a ``@pytest.mark.metadata(blockers=[BZ(<bug_id>])`` decorator
+  to you test case. At run time, the property ``BZ(<bug_id>).blocks`` will be checked,
+  if ``True``, the test case will be skipped. For example,
+
```

Comment:
Can you also add when `BZ().blocks` will return True/False?
It checks the status of BZ and then decides if it blocks or not, so can you mention the states for which it will return True/False?

Reply:
Agree, can you add that this statement depends on the appliance version?

Reply:
Can you add here explanation about `forced_streams`?

Reply:
I would also add that if BZ blocks only one check in the test case, it's better to do a workaround with `BZ(<bug_id>).blocks` then blocking the whole test case